responsibleai/responsibleai/_managers/explainer_manager.py

```diff
@@ -342,6 +342,7 @@ def _load(path, rai_insights):
         meta = json.loads(meta)
         inst.__dict__['_' + IS_RUN] = meta[IS_RUN]
 
+        inst.__dict__[EXPLANATION] = None
```

Comment:
nit: I think it's a tiny bit cleaner to do:

```
if explanation_path.exists():
   ...
else:
   inst.__dict__[EXPLANATION] = None
```

Reply:
If you want I can change, but I actually prefer setting the default every time, and updating it if required

Reply:
ok!  It just seems redundant, since it's set right on the line below again in the if statement.  It's a minor style thing.